File: core/tests/fixtures/mothers/base/mother_mapping.py
from enum import Enum


class MotherMapping(Enum):
    """Enum class for mapping mother data types to their corresponding Python types."""
    NONE = 'mm:none'
    ARRAY = 'mm:array'
    DATE = 'mm:date'
    REQUIRED = 'mm:required'
    MOTHER = 'mm:mother'

File: editor/tests/fixtures/mothers/base/mother-mapping.ts
export enum MotherMapping {
  NONE = 'mm:none',
  UUID = 'mm:uuid',
  ARRAY = 'mm:array',
  DATE = 'mm:date',
  REQUIRED = 'mm:required',
  MOTHER = 'mm:mother',
}

export const isMotherMapping = (value: string): value is MotherMapping => {
  return Object.values(MotherMapping).includes(value as MotherMapping);
};
